src/components/ToolButton/ToolButton.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useRef, useEffect } from 'react';
+import { ToolType } from 'canvas/Enums';
+import { ActionButton } from 'components/ActionButton/ActionButton';
+import './ToolButton.css';
+
+interface ToolButtonProps {
+    toolType: ToolType;
+    selectedType: ToolType;
+    setSelectedType: React.Dispatch<React.SetStateAction<ToolType>>;
+}
+
+export const ToolButton: React.FC<ToolButtonProps> = ( {toolType, selectedType, setSelectedType, ...props }) => {    
+    const setCanvasToolType = new CustomEvent('setToolType', {
+        detail: { toolType: toolType }
+    });
+
+    const canvasRef = useRef(document.querySelector('canvas'));
+    useEffect(() => {
+        canvasRef.current = document.querySelector('canvas');
+    }, [canvasRef]);
+
+    let toolBorder = {};
+    if (selectedType === toolType) {
+        toolBorder = {
+            border: '#707070 dashed .5px',
+            borderRadius: '15px'
```

Comment:
I see tiny differences in the placement of the canvas and the toolButtons when I toggle between selecting the straightline tool and the freelinetool. My best guess is adding this border shifts other elements around a tiny bit, but it might also be something else.

Reply:
This should be fixed now. Thanks!